core/types/transaction_signing.go

```diff
@@ -108,6 +114,37 @@ type Signer interface {
 	Equal(Signer) bool
 }
 
+// EIP86Signer implements the the abstract signing feature
+type EIP86Signer struct {
+	EIP155Signer
+}
+
+func NewEIP86Signer(chainId *big.Int) EIP86Signer {
+	return EIP86Signer{
+		EIP155Signer: NewEIP155Signer(chainId),
+	}
+}
+
+func (s EIP86Signer) Equal(s2 Signer) bool {
+	_, ok := s2.(EIP86Signer)
+	return ok && s.EIP155Signer.Equal(s2)
+}
+
+// isAbstractSigner checks whether the transaction signature is that of
+// the abstract signer. The abstract signer is determined by V being equal
+// to the chain identifier and r and s being 0.
+func isAbstractSigner(tx *Transaction, chainId *big.Int) bool {
+	v, r, s := tx.RawSignatureValues()
+	return v.Cmp(chainId) == 0 && r.BitLen() == 0 && s.BitLen() == 0
```

Comment:
A requirement for it being an abstract-signer tx, is that `gasprice = 0, nonce = 0, value = 0`. Is this checked anywhere?  

[spec](https://github.com/ethereum/EIPs/blob/94be170f814a9ceadbf82a2728e568b26e713726/EIPS/abstraction.md)